cli/tests/integrations/test_task.py

```diff
@@ -132,11 +130,11 @@ def test_log_pod_task():
             ['dcos', 'task', 'log', 'good-container', 'stderr'])
 
         # pod task log are not executor logs, so normal executor stderr
-        # logs shouldn't be seen and this pod shoudn't have any logging
+        # logs shouldn't be seen and this pod shouldn't have any logging
         # to stderr
-        assert returncode == 1
-        assert stderr == b'No files exist. Exiting.\n'
-        assert stdout == b''
+        assert returncode == 0
+        assert stderr == b''
```

Comment:
A `stderr` file is now being created for each pod task, so the command no longer fails to find it. The file stays empty, so it can still be distinguished from the executor's `stderr`. I'm not sure why `stdout` now has the newline, maybe because the command succeeds instead of failing? As the comment says, it's really the value of `stderr` that's the key assertion of this test.